App/HejDTO/Sources/HejDTO/Contests/GET/ContestWinner/ContestWinners.swift

```diff
@@ -0,0 +1,18 @@
+
+import Foundation
+
+public struct ContestWinners: Codable, Equatable {
```

Comment:
should it be a `winners` struct inside the `Contests` namespace?

Reply:
Moved :D